apisix/admin/stream_routes.lua

```diff
@@ -142,8 +163,18 @@ function _M.delete(id)
         return 400, {error_msg = "missing stream route id"}
     end
 
+    local items,_ = routes()
     local key = "/stream_routes/" .. id
     -- core.log.info("key: ", key)
+    local refer_list = {}
+    if items ~= nil then
+        refer_list=check_router_refer(items,id)
+    end
+    if #refer_list >0 then
+        local warn_message = key.." is referred by "..table.concat(refer_list,";;")
+        return 400,warn_message
+    end
```

Comment:
After thinking about it for a while, it seems that the `refer_list` lifecycle is only used here. Do we use `core.tablepool` to fetch and release `refer_list`?

Reply:
Thanks for your comments about core.tablepool.